windows/nearobject/winrt/windows.devices.nearobject/pch.cpp

```diff
@@ -0,0 +1 @@
+﻿#include "pch.h"
```

Comment:
IIRC, you're not a big fan of precompiled headers. Is this something that is required by winrt?

Reply:
I figured out how to do disable use of PCH in `cppwinrt.exe`; there's a switch `-pch <pch filename>` that can be specified, and using `'.'` as the value disables PCH entirely. In doing so, I figured out we absolutely should not disable PCH in the generated projection bindings 😄

The generated WinRT headers (`winrt/Windows.Foundation.h, winrt/Windows.Foundation.Collections.h`) are massive and pretty complex. Some of them are several MBs in size. This is exactly the situation where PCH provides benefit since we don't want the compiler processing several MB headers for each compilation unit. So, I think we should keep the PCH and document that it should be used sparingly, possibly only for C++/WinRT projection bindings.

Reply:
Documented the restrictions on PCH use.